### PR TITLE
Fixing https://github.com/wso2/product-apim/issues/10154

### PR DESCRIPTION
**Pull request by chanaka3d**
Fixing https://github.com/wso2/product-apim/issues/10154 and devportal responsive issues.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx [hunk 1]
```diff
@@ -347,7 +354,7 @@ class Details extends React.Component {
     handleDrawerClose() {
         this.setState({ open: false });
         const user = AuthManager.getUser();
-        if(user != null){
+        if (user != null) {
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx:357]: This should be `!==`

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx [hunk 2]
```diff
@@ -74,8 +92,17 @@ const styles = (theme) => ({
...
+        [theme.breakpoints.down('sm')]: {
+            display: 'none !important',
+        }
+    },
     submenu: {
-        paddingLeft: theme.spacing(4),
+        paddingLeft: 12,
+        [theme.breakpoints.down('sm')]: {
+            paddingLeft: 0,
+            color: '#cccccc',
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx:104]: Shouldn't it come from theme ?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx [hunk 3]
```diff
@@ -122,78 +149,84 @@ function LeftMenuItem(props) {
...
+                    classes.leftLInk,
+                    {
+                        [classes.leftLink_IconLeft]: leftMenu === 'icon left',
+                        [classes.submenu]: submenu,
+                    },
+                    'leftLInk',
+                )}
+                to={to}
+                style={{ backgroundColor: activeBackground }}
+                title={props.text}
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx:164]: IMO it's better to do props destructuring at the top level, Should have caught by eslint :/

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx (hunk 1 above)
Comment (chanaka3d) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx:362]: done

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx (hunk 2 above)
Comment (chanaka3d) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx]: done

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx (hunk 1 above)
Comment (chanaka3d) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx:362]: done

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx (hunk 3 above)
Comment (chanaka3d) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/LeftMenuItem.jsx]: done

**Review by tmkasun — APPROVED** [review]

**Pull request merged**